test/test_wave.py

```diff
@@ -78,89 +79,149 @@ def mesh_factory(n):
-        sym_f = sym.diff(sym_t)(sym.diff(sym_t)(sym_phi)) - c**2\
-                    * sym.div(sym.grad(dim, sym_phi))
+    # Check order of accuracy
 
-        # u = phi_t
-        sym_u = sym.diff(sym_t)(sym_phi)
+    from pytools.convergence import EOCRecorder
+    eoc_rec = EOCRecorder()
 
-        # v = c*grad(phi)
-        sym_v = [c * sym.diff(sym_coords[i])(sym_phi) for i in range(dim)]
+    for n in [4, 8, 16]:
+        mesh = mesh_factory(n)
+
+        from grudge.eager import EagerDGDiscretization
+        discr = EagerDGDiscretization(cl_ctx, mesh, order=order)
 
-        # rhs(u part) = c*div(v) + f
-        # rhs(v part) = c*grad(u)
-        sym_rhs = flat_obj_array(
-                    c * sym.div(sym_v) + sym_f,
-                    make_obj_array([c]) * sym.grad(dim, sym_u))
+        nodes = discr.nodes().with_queue(queue)
 
         def sym_eval(expr, t):
             return sym.EvaluationMapper({"x": nodes, "t": t})(expr)
 
-        u = sym_eval(sym_u, 0.)
-        v = sym_eval(sym_v, 0.)
-        f = sym_eval(sym_f, 0.)
+        t_check = 1.23456789
+
+        u = sym_eval(sym_u, t_check)
+        v = sym_eval(sym_v, t_check)
 
-        from mirgecom.wave import wave_operator
-        rhs = wave_operator(discr, c=c, w=flat_obj_array(u, v))
-        rhs[0] = rhs[0] + f
+        fields = flat_obj_array(u, v)
 
-        expected_rhs = sym_eval(sym_rhs, 0.)
+        rhs = wave_operator(discr, c=c, w=fields)
+        rhs[0] = rhs[0] + sym_eval(sym_f, t_check)
 
-        err = np.max(np.array([la.norm((rhs[i] - expected_rhs[i]).get(), np.inf)
-                for i in range(dim+1)]))
-        eoc_rec.add_data_point(1./n, err)
+        expected_rhs = sym_eval(sym_rhs, t_check)
+
+        eoc_rec.add_data_point(1./n, max_inf_norm(rhs - expected_rhs))
 
         if visualize:
             from grudge.shortcuts import make_visualizer
             vis = make_visualizer(discr, discr.order)
-            vis.write_vtk_file("result_{n}.vtu".format(n=n),
-                    [
-                        ("u", u),
-                        ("v", v),
-                        ("rhs_u_actual", rhs[0]),
-                        ("rhs_v_actual", rhs[1:]),
-                        ("rhs_u_expected", expected_rhs[0]),
-                        ("rhs_v_expected", expected_rhs[1:]),
+            vis.write_vtk_file("wave_accuracy_{order}_{n}.vtu".format(order=order,
+                        n=n), [
+                            ("u", fields[0]),
+                            ("v", fields[1:]),
+                            ("rhs_u_actual", rhs[0]),
+                            ("rhs_v_actual", rhs[1:]),
+                            ("rhs_u_expected", expected_rhs[0]),
+                            ("rhs_v_expected", expected_rhs[1:]),
                         ])
 
     print("Approximation error:")
     print(eoc_rec)
     assert(eoc_rec.order_estimate() >= order - 0.5 or eoc_rec.max_error() < 1e-11)
+
+    # Check stability
```

Comment:
Could you make stability a separate test?

Reply:
Done.